online-tutorial-series/03_computation.ipynb

```diff
@@ -0,0 +1,330 @@
+{
```

Comment:
<p>Hoyer... (I've made this mistake before :D)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>